app/models/facility.rb

```diff
@@ -43,6 +43,10 @@ def accounts
   validates_format_of    :abbreviation, with: /^[a-zA-Z\d\-\.\s]+$/, message: "may include letters, numbers, hyphens, spaces, or periods only"
   validates_format_of    :journal_mask, with: /^C\d{2}$/, message: "must be in the format C##"
 
+  validates :order_notification_recipient,
+    format: { with: /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,6}$/i },
```

Comment:
This is lifted from a validation in the `User` model. Do we want to extract a validator, or use something like https://github.com/balexand/email_validator ?